swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -380,6 +424,15 @@ public void setModel(ComboBoxModel<SSListItem> _model) {
 				: null;
 	}
 
+	/**
+	 * create SSBaseComboBox
+	 */
+	public SSBaseComboBox() {
```

Comment:
Was thinking this was adding an empty constructor, but then remembered this is an abstract class.

Reply:
That's what I meant by empty constructor, but that might not be the right terminology. Doesn't matter as it should be a non-issue.